<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>782</width>
    <height>936</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <widget class="QGroupBox" name="automayicScanGroupBox">
      <property name="title">
       <string>Automatic Scan </string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="3">
        <widget class="QDoubleSpinBox" name="percentileMaxDoubleSpinBox">
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="suffix">
          <string>%</string>
         </property>
         <property name="minimum">
          <double>0.000000000000000</double>
         </property>
         <property name="maximum">
          <double>100.000000000000000</double>
         </property>
         <property name="value">
          <double>100.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="0" column="4">
        <widget class="QDoubleSpinBox" name="counterMaxDoubleSpinBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="mouseTracking">
          <bool>true</bool>
         </property>
         <property name="acceptDrops">
          <bool>true</bool>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="accelerated">
          <bool>true</bool>
         </property>
         <property name="suffix">
          <string/>
         </property>
         <property name="minimum">
          <double>-1000000000.000000000000000</double>
         </property>
         <property name="maximum">
          <double>1000000000.000000000000000</double>
         </property>
         <property name="value">
          <double>100000.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="3">
        <widget class="PlotWidget" name="xyScanView"/>
       </item>
       <item row="1" column="3" colspan="2">
        <widget class="PlotWidget" name="contrastView"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="axisLabel">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Axis</string>
         </property>
         <property name="scaledContents">
          <bool>false</bool>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
         <property name="indent">
          <number>0</number>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="amplitudeLabel">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Amplitude (V)</string>
         </property>
         <property name="scaledContents">
          <bool>false</bool>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
         <property name="indent">
          <number>0</number>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QLabel" name="frequencyLabel">
         <property name="text">
          <string>Frequency (Hz)</string>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="QDoubleSpinBox" name="percentileMinDoubleSpinBox">
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="suffix">
          <string>%</string>
         </property>
         <property name="minimum">
          <double>0.000000000000000</double>
         </property>
         <property name="maximum">
          <double>100.000000000000000</double>
         </property>
         <property name="value">
          <double>0.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="2" column="4">
        <widget class="QDoubleSpinBox" name="counterMinDoubleSpinBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="mouseTracking">
          <bool>true</bool>
         </property>
         <property name="acceptDrops">
          <bool>true</bool>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="accelerated">
          <bool>true</bool>
         </property>
         <property name="suffix">
          <string/>
         </property>
         <property name="minimum">
          <double>-1000000000.000000000000000</double>
         </property>
         <property name="maximum">
          <double>10000000.000000000000000</double>
         </property>
         <property name="value">
          <double>0.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="xAxisLabel">
         <property name="text">
          <string>X</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QDoubleSpinBox" name="xAmplitudeDoubleSpinBox">
         <property name="decimals">
          <number>0</number>
         </property>
         <property name="minimum">
          <double>1.000000000000000</double>
         </property>
         <property name="maximum">
          <double>70.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QDoubleSpinBox" name="xFrequencyDoubleSpinBox">
         <property name="decimals">
          <number>0</number>
         </property>
         <property name="minimum">
          <double>1.000000000000000</double>
         </property>
         <property name="maximum">
          <double>2000.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="3" column="3">
        <widget class="QRadioButton" name="peercentiesRadioButton">
         <property name="text">
          <string>Percentiles</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="4">
        <widget class="QRadioButton" name="manualRadioButton">
         <property name="text">
          <string>Counter/s</string>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="yAxisLabel">
         <property name="text">
          <string>Y</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QDoubleSpinBox" name="yAmplitudeDoubleSpinBox">
         <property name="decimals">
          <number>0</number>
         </property>
         <property name="minimum">
          <double>1.000000000000000</double>
         </property>
         <property name="maximum">
          <double>70.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="QDoubleSpinBox" name="yFrequencyDoubleSpinBox">
         <property name="decimals">
          <number>0</number>
         </property>
         <property name="minimum">
          <double>1.000000000000000</double>
         </property>
         <property name="maximum">
          <double>2000.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="zAxisLabel">
         <property name="text">
          <string>Z</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QDoubleSpinBox" name="zAmplitudeDoubleSpinBox">
         <property name="decimals">
          <number>0</number>
         </property>
         <property name="minimum">
          <double>1.000000000000000</double>
         </property>
         <property name="maximum">
          <double>70.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="5" column="2">
        <widget class="QDoubleSpinBox" name="zFrequencyDoubleSpinBox">
         <property name="decimals">
          <number>0</number>
         </property>
         <property name="minimum">
          <double>1.000000000000000</double>
         </property>
         <property name="maximum">
          <double>2000.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="6" column="0" colspan="3">
        <widget class="QPushButton" name="startScanPushButton">
         <property name="text">
          <string>Start XY Scan</string>
         </property>
        </widget>
       </item>
      </layout>
      <zorder>xyScanView</zorder>
      <zorder>contrastView</zorder>
      <zorder>startScanPushButton</zorder>
      <zorder>axisLabel</zorder>
      <zorder>xAxisLabel</zorder>
      <zorder>yAxisLabel</zorder>
      <zorder>amplitudeLabel</zorder>
      <zorder>xAmplitudeDoubleSpinBox</zorder>
      <zorder>yAmplitudeDoubleSpinBox</zorder>
      <zorder>counterMinDoubleSpinBox</zorder>
      <zorder>percentileMinDoubleSpinBox</zorder>
      <zorder>manualRadioButton</zorder>
      <zorder>peercentiesRadioButton</zorder>
      <zorder>counterMaxDoubleSpinBox</zorder>
      <zorder>percentileMaxDoubleSpinBox</zorder>
      <zorder>xFrequencyDoubleSpinBox</zorder>
      <zorder>yFrequencyDoubleSpinBox</zorder>
      <zorder>frequencyLabel</zorder>
      <zorder>zFrequencyDoubleSpinBox</zorder>
      <zorder>zAxisLabel</zorder>
      <zorder>zAmplitudeDoubleSpinBox</zorder>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QGroupBox" name="groupBox_3">
      <property name="title">
       <string>Manual Scan</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0" rowspan="2" colspan="2">
        <widget class="QLabel" name="xAxisLabel_2">
         <property name="text">
          <string>X</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2" rowspan="2">
        <widget class="QPushButton" name="stepXBackwardPushButton">
         <property name="text">
          <string>Step Backward </string>
         </property>
        </widget>
       </item>
       <item row="0" column="3" rowspan="2">
        <widget class="QPushButton" name="stepXForwardPushButton">
         <property name="text">
          <string>Step Forward </string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QCheckBox" name="xAxisCheckBox">
         <property name="text">
          <string>Output</string>
         </property>
        </widget>
       </item>
       <item row="1" column="4">
        <widget class="QScrollBar" name="xAxisScrollBar">
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="pageStep">
          <number>1</number>
         </property>
         <property name="value">
          <number>50</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="2" column="0" rowspan="2" colspan="2">
        <widget class="QLabel" name="yAxisLabel_2">
         <property name="text">
          <string>Y</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2" rowspan="2">
        <widget class="QPushButton" name="stepYBackwardPushButton">
         <property name="text">
          <string>Step Backward</string>
         </property>
        </widget>
       </item>
       <item row="2" column="3" rowspan="2">
        <widget class="QPushButton" name="stepYForwardPushButton">
         <property name="text">
          <string>Step Forward </string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QCheckBox" name="yAxisCheckBox">
         <property name="text">
          <string>Output</string>
         </property>
        </widget>
       </item>
       <item row="3" column="4">
        <widget class="QScrollBar" name="yAxisScrollBar">
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="pageStep">
          <number>1</number>
         </property>
         <property name="value">
          <number>50</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="4" column="0" rowspan="2" colspan="2">
        <widget class="QLabel" name="zAxisLabel_2">
         <property name="text">
          <string>Z</string>
         </property>
        </widget>
       </item>
       <item row="4" column="2" rowspan="2">
        <widget class="QPushButton" name="stepZBackwardPushButton">
         <property name="text">
          <string>Step Backward</string>
         </property>
        </widget>
       </item>
       <item row="4" column="3" rowspan="2">
        <widget class="QPushButton" name="stepZForwardPushButton">
         <property name="text">
          <string>Step Forward </string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QCheckBox" name="zAxisCheckBox">
         <property name="text">
          <string>Output</string>
         </property>
        </widget>
       </item>
       <item row="5" column="4">
        <widget class="QScrollBar" name="zAxisScrollBar">
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="pageStep">
          <number>1</number>
         </property>
         <property name="value">
          <number>50</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>782</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QGraphicsView</extends>
   <header>pyqtgraph</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
